### DISCUSSION TITLE
Why doesn't Weaviate database use the latest version, still using version 1.19, and does Dify currently support higher versions of Weaviate?General Discussion

### DISCUSSION THREAD

**Discussion by daizzhisheng** [langgenius/dify#7692] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Why doesn't Weaviate database use the latest version, still using version 1.19, and does Dify currently support higher versions of Weaviate?

**Comment by asrguru20** [comment]
I myself upgraded manually it worked
